course/include/isometry.h

```diff
@@ -0,0 +1,78 @@
+#ifndef ISOMETRY_H
+#define ISOMETRY_H
+
+#include <cstdint>
+#include <iostream>
+#include <ostream>
+
+namespace ekumen {
+namespace math {
+
+class Vector3{
+    public:
+        // Constructors
+        Vector3() : x_(0), y_(0), z_(0) {};
```

Comment:
Exploit delegating constructors, avoid replicating the same initialization in more than one place.